.github/workflows/functional-test-noncloud.yaml

```
@@ -361,7 +367,6 @@ jobs:
           # The functional test is designed to use default namespace. So you must create the environment for default namespace.
           rad env create kind-radius --namespace default
           rad env switch kind-radius
-        continue-on-error: true  
```

I'd like some feedback from a few people on this. Why are we not failing fast?